spec/warden/jwt_auth/token_encoder_spec.rb

```diff
@@ -65,5 +65,18 @@
         expect(decoded_payload['foo']).to eq('bar')
       end
     end
+
+    context 'with issuer claim' do
+      let(:issuer) { 'http://example.com' }
+      before do
+        Warden::JWTAuth.configure do |config|
+          config.issuer = issuer
+        end
+      end
```

Comment:
Same. Let's use shared config.

Reply:
added in 508aa17f15075a91b5ed3b3623a4a5892c7b37ff